test/ext/test_DataInterpolations.jl

```diff
@@ -35,7 +35,8 @@ struct InterpolationTest{N,I<:AbstractInterpolation} # N = output dim. of interp
 end
 function InterpolationTest(
     interp::I; is_der1_zero=false, is_der2_zero=false
-) where {T,N,I<:AbstractInterpolation{T,N}}
+) where {T,I<:AbstractInterpolation{T}}
+    N = get_output_dim(interp.u)
```

Comment:
```suggestion
) where {T,N,I<:AbstractInterpolation{T}}
```

Wouldn't this work?

Reply:
I tried but it didn't, no, complaining about unused type parameters.